submissions/Barasii/a-tiny-JS-world-OOP/index.js

```diff
@@ -0,0 +1,86 @@
+class Inhabitant {
+   constructor(species, name, gender, legs, hands, saying, friends) {
+      this.species = species;
+      this.name = name;
+      this.gender = gender;
+      this.legs = legs;
+      this.hands = hands;
+      this.saying = saying;
+      this.friends = [];
+   }
+
+   toString() {
+      return Object.keys(this).map(prop => `${this[prop]}`).join('; ');
+      }
```

Comment:
Consistent code style is important. It makes reading easier without eye unconsciously distracted by tiny formatting bugs.
Curly braces indentation is expected to tell something about code structure. Wrong indentation makes a reader to check if structure/nesting is OK.

Take care of your readers. Code is written for other people.

Reply:
I reinstalled Prettier